app/nasa_collaboration/page.js

```diff
@@ -5,13 +5,23 @@ import styles from './page.module.css';
 
 // Read "/app/nasa_collaboration/README.md" for more info about the API_KEY
 // You need a proper API_KEY for the requests to work
-const API_KEY = 'API_KEY';
+const API_KEY = "eIHrNKxbUnq1lNBHdoSMLh6PctlNbq8aXADnLASd";
```

Comment:
For security, it is advised not to leave API keys in code that is committed to a repository.
One solution is using a .env file that is then added to your Git ignore list. See https://medium.com/@rivaifnasution/tutorial-how-to-use-dotenv-in-react-for-beginners-ac79d9b6ddbf